src/modules/getData.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable no-undef */
+export const main = document.querySelector('.container');
+
+const getData = () => {
```

Comment:
You seem to reuse this function even though it has already been declared in the main `index.js` file. Be sure to just call for it on your main file, as you have already created it on a separate file.

Reply:
It's a bit difficult to make changes to this function because, when I tried it gave a lot of errors about the node. So consider this for me, please! because it's crucial to the application to work properly.